src/plot_api/plot_schema.js

```diff
@@ -208,6 +208,142 @@ exports.findArrayAttributes = function(trace) {
     return arrayAttributes;
 };
 
+/*
```

Comment:
:sos: semi-useless lint comment alert :sos: 

jsDOC is enabled on comments starting with `/**` (see http://usejsdoc.org/about-getting-started.html). As we don't extract those jsDOC comment (yet), adding this extra `*` would _just_ make that comment look prettier in my editor. Non-blocking.